internal/functions/deploy/bundle.go

```diff
@@ -122,41 +122,24 @@ func GetBindMounts(cwd, hostFuncDir, hostOutputDir, hostEntrypointPath, hostImpo
 			binds = append(binds, hostOutputDir+":"+dockerOutputDir+":rw")
 		}
 	}
-	// Allow entrypoints outside the functions directory
-	hostEntrypointDir := filepath.Dir(hostEntrypointPath)
-	if len(hostEntrypointDir) > 0 {
-		if !filepath.IsAbs(hostEntrypointDir) {
-			hostEntrypointDir = filepath.Join(cwd, hostEntrypointDir)
-		}
-		if !strings.HasSuffix(hostEntrypointDir, sep) {
-			hostEntrypointDir += sep
-		}
-		if !strings.HasPrefix(hostEntrypointDir, hostFuncDir) &&
-			!strings.HasPrefix(hostEntrypointDir, hostOutputDir) {
-			dockerEntrypointDir := utils.ToDockerPath(hostEntrypointDir)
-			binds = append(binds, hostEntrypointDir+":"+dockerEntrypointDir+":ro")
-		}
-	}
```

Comment:
This is no longer needed because we entrypoint path is included when walking import paths.